<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro_de_ONT</class>
 <widget class="QDialog" name="Registro_de_ONT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de ONT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../assets/icons/ONT.png</normaloff>../../assets/icons/ONT.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Registro_de_ONT{
background-color: qlineargradient(spread:pad, x1:0.488444, y1:0.392, x2:0.483241, y2:1, stop:0.266917 rgba(1, 59, 61, 255), stop:1 rgba(255, 255, 255, 244));}
</string>
  </property>
  <widget class="QPushButton" name="btn_codebar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>23</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Leer etiqueta&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 59, 61);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../assets/icons/camara.svg</normaloff>../../assets/icons/camara.svg</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guardar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>23</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Guardar ONT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 59, 61);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../assets/icons/Guardar.svg</normaloff>../../assets/icons/Guardar.svg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>150</y>
     <width>130</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Número se Serie:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>190</y>
     <width>130</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Modelo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>230</y>
     <width>130</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Ubicación:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Expediente Técnico:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_regresar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>23</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Regresar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 0, 0);
color:rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../assets/icons/atras bc.svg</normaloff>../../assets/icons/atras bc.svg</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_ns">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Número de Serie</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_modelo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Ingresar Modelo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="str_ubicacion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 10px;</string>
   </property>
   <item>
    <property name="text">
     <string>Almacén</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>En campo</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_imagen">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>23</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Importar etiqueta&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 59, 61);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../assets/icons/foto.svg</normaloff>../../assets/icons/foto.svg</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>52</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Área:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="str_area">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Seleccionar Área&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 10px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Copé:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="str_cope">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Seleccionar Copé&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 10px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Titulo">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Comercilazadora LECE Comunicaciones&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Times New Roman&quot;;
color:rgba(239, 191, 188, 255);
</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>Comercilazadora LECE Comunicaciones</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot; rgb(255, 255, 255);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Registro de O.N.T.</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label_2</zorder>
   <zorder>Titulo</zorder>
  </widget>
  <widget class="QComboBox" name="str_exptec">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Seleccionar Expediente Técnico&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imagen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_imagen</zorder>
  <zorder>btn_codebar</zorder>
  <zorder>btn_guardar</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>ln_ns</zorder>
  <zorder>ln_modelo</zorder>
  <zorder>str_ubicacion</zorder>
  <zorder>btn_imagen</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>str_exptec</zorder>
  <zorder>btn_regresar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
